components/lib/VBlurb.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <v-stack direction="column" align="center" style="text-align: center">
-    <v-image v-if="image" :src="image" :alt="alt" />
+    <v-image v-if="image" path="image" :alt="alt" />
```

Comment:
image is passed as prop to VBlurb
```suggestion
    <v-image v-if="image" :path="image" :alt="alt" />
```

Reply:
Oops 🤦‍♂️, good catch!